doc/timers.adoc

```diff
@@ -35,7 +35,7 @@ Each task gets a timer. The timer has three properties:
 - A _notification set._
 
 At periodic intervals (ticks), if the enable bit is set, the kernel checks the
-deadline to see if it is in the future. If not (it is `<=` the current time),
+deadline to see if it is in the future. If not (it is &#2264; the current time),
```

Comment:
FWIW, `<=` is a Rust operator and I'm more comfortable with that in the document than the Unicode escape for the math equivalent. Personal preference thing.

Reply:
The problem here is that asciidoc turned it into a double arrow!

Reply:
@riking might have noticed that the Rust operator `<=` is being rendered as <code>&#8656;</code> (`&#8656;`) which is `U+21D0`.

This is probably because GitHub renders with `asciidoctor` rather than `asciidoc`:
```
# What I think you prefer (with asciidoc):
❯ echo '`<=`' | asciidoc -s -
<div class="paragraph"><p><code>&lt;=</code></p></div>

# What you're currently getting (with asciidoctor):
❯ echo '`<=`' | asciidoctor -s -
<div class="paragraph">
<p><code>&#8656;</code></p>
</div>

# How we can fix it:
❯ echo '`\<=`' | asciidoctor -s -
<div class="paragraph">
<p><code>&lt;=</code></p>
</div>

```
We can fix the Rust operator and prevent the [character replacement substitution](https://docs.asciidoctor.org/asciidoc/latest/subs/replacements/) by using a backslash to escape the sequence, like this: `\<=`.

Reply:
Ah! I misunderstood. Thank you @riking and @fernandogmo for walking me through that. Let's escape it as in that last suggestion.

I'm...not sure if I can edit your PR @riking? So if you'd be willing to do the `\<=` thing I'll merge.

Reply:
Done